src/main/java/springboot/webapp/usersmanager/Services/UserService.java

```diff
@@ -0,0 +1,21 @@
+package springboot.webapp.usersmanager.Services;
+
+import springboot.webapp.usersmanager.entities.User;
+
+import java.util.List;
+
+public interface UserService {
+    List<User> getAll();
+
+    void save(User user);
+
+    void update(User user, int userId);
```

Comment:
why userId can't be part of the User object? You can combine save and update and put the name like "put" or "upsert"

Reply:
user was taked from request body without field id, field user id we take from url. We should to update user with id that was passed from url